src/enzo/RebuildHierarchy.C

```diff
@@ -395,6 +402,19 @@ int RebuildHierarchy(TopGridData *MetaData,
       tt1 = ReturnWallTime();
       RHperf[3] += tt1-tt0;
 
+      /* Deposit flagging field for active particles.
+
+     NJG:  This is a hack so that the accretion zone of sink particles are
+           resolved at all times.  While this will work fine for small number of
+           particles, there will be scaling issues if one tries to do nonlocal
+           refinement on large numbers of particles
```

Comment:
After how many sink particles do the scaling issues become important? This would be good to include as a warning in the documentation.

Reply:
Not sure. This could be difficult to determine and would also depend on the specific problem. This may be a good point to raise at the meeting or on the slack channel.

Reply:
I agree with @fearmayo and it's unclear what's the limit. Nobody's tried to push the limit. It's definitely a good idea to put a warning in the documentation. I'll add this.

Reply:
Actually, there is no dev documentation for the ActiveParticles. Issue #75